.pre-commit-config.yaml

```
@@ -89,6 +89,9 @@ repos:
             # No direct imports from conftest
             conftest\ import|
             import\ conftest
+
+            # Check for use of pandas.testing instead of tm
+            pd.testing.\
```

You need to escape the dots
```suggestion
            pd\.testing\.
```